lumicks/pylake/channel.py

```diff
@@ -348,6 +393,9 @@ def __init__(self, data, start, dt):
         self.stop = start + len(data) * dt
         self.dt = dt
 
+    def _with_data_source(self, data):
```

Comment:
I feel this would be just called `_with_data`.
Unless it already has another meaning or is confusing in the context of pylake of Python in general...

Reply:
Sounds good! Amended! :)